src/transformers/generation/logits_process.py

```diff
@@ -133,14 +133,52 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 class MinNewTokensLengthLogitsProcessor(LogitsProcessor):
     r"""
     [`LogitsProcessor`] enforcing a min-length of new tokens by setting EOS (End-Of-Sequence) token probability to 0.
+    Note that for decoder-only models, such as Llama2, `min_length` will compute the length of `prompt + newly
+    generated tokens` whereas for other models it will behave as `min_new_tokens`,that is, taking only into account the
```

Comment:
ultra nit 
```suggestion
    generated tokens` whereas for other models it will behave as `min_new_tokens`, that is, taking only into account the
```